<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>questionDialog</class>
 <widget class="QDialog" name="questionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,15">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>目前还剩</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_count">
         <property name="styleSheet">
          <string notr="true">
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>道题</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="title">
       <property name="styleSheet">
        <string notr="true">font: 32pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="opt1">
         <property name="styleSheet">
          <string notr="true">font: 20pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="opt2">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
font: 20pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="opt3">
         <property name="styleSheet">
          <string notr="true">font: 20pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="opt4">
         <property name="styleSheet">
          <string notr="true">font: 20pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="opt5">
         <property name="styleSheet">
          <string notr="true">font: 20pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
